tests/cases/conformance/jsdoc/jsdocTemplateTag5.ts

```diff
@@ -0,0 +1,70 @@
+// @noEmit: true
+// @allowJs: true
+// @checkJs: true
+// @strict: true
+// @Filename: a.js
+
+/**
+ * Should work for function declarations
+ * @constructor
+ * @template {string} K
+ * @template V
+ */
+function Multimap() {
+    /** @type {Object<string, V>} TODO: Remove the prototype from the fresh object */
+    this._map = {};
+};
+
+Multimap.prototype = {
+    /**
+     * @param {K} key the key ok
+     * @returns {V} the value ok
+     */
+    get(key) {
```

Comment:
only works for methods, not properties with function values. That is, `get: function(key) ...` doesn't work